src/test/java/com/pba/authservice/integration/ActiveUserDaoIntegrationTest.java

```diff
@@ -0,0 +1,177 @@
+    }
+
+    @Test
+    public void testSaveActiveUser() throws IllegalAccessException {
+        // given
+        ActiveUser activeUser = activeUserMockGenerator.generateMockActiveUser();
+
+        // when
+        ActiveUser result = activeUserDao.save(activeUser);
+
+        // then
+        Assertions.assertEquals(activeUser.getUid(), result.getUid());
+    }
+
+    private void addMockListOfActiveUsers(List<ActiveUser> activeUserList) throws IllegalAccessException {
+        for (ActiveUser activeUser : activeUserList) {
+            activeUserDao.save(activeUser);
+        }
+    }
+
+    private List<UUID> extractUids(List<ActiveUser> activeUserList) {
+        return activeUserList.stream().map((activeUser -> activeUser.getUid())).collect(Collectors.toList());
+    }
+    @Test
+    public void testGetAllActiveUsers() throws IllegalAccessException {
+        // given
+        List<ActiveUser> activeUserList = activeUserMockGenerator.generateMockListOfActiveUsers(10);
+        this.addMockListOfActiveUsers(activeUserList);
+        List<UUID> activeUsersUids = this.extractUids(activeUserList);
+
+        // when
+        List<ActiveUser> result = activeUserDao.getAll();
+        List<UUID> resultUids = this.extractUids(result);
+
+        // then
+        Assertions.assertEquals(activeUsersUids, resultUids);
+    }
+
+    @Test
+    public void testGetPresentActiveUserByUid() throws IllegalAccessException {
+        // given
+        ActiveUser activeUser = activeUserMockGenerator.generateMockActiveUser();
+        activeUserDao.save(activeUser);
+
+        // when
+        Optional<ActiveUser> result = activeUserDao.getById(activeUser.getUid());
+
+        // then
+        Assertions.assertEquals(activeUser.getUid(), result.get().getUid());
+    }
+
+    @Test
+    public void testGetEmptyActiveUserByUid() throws IllegalAccessException {
+        // given
+        UUID uid = UUID.randomUUID();
+
+        // when
+        Optional<ActiveUser> result = activeUserDao.getById(uid);
+
+        // then
+        Assertions.assertTrue(result.isEmpty());
+    }
+
+    @Test
+    public void testDeletePresentActiveUserByUid() throws AuthDaoException, IllegalAccessException {
+        // given
+        ActiveUser activeUser = activeUserMockGenerator.generateMockActiveUser();
+        activeUserDao.save(activeUser);
+
+        // when
+        ActiveUser result = activeUserDao.deleteById(activeUser.getUid());
+
+        // then
+        Assertions.assertEquals(activeUser.getUid(), result.getUid());
+        Assertions.assertEquals(0, activeUserDao.getAll().size());
+    }
+
+    @Test
+    public void testDeleteEmptyActiveUserByUid() throws IllegalAccessException {
+        // given
+        UUID uid = UUID.randomUUID();
+
+        try {
+            // when
+            activeUserDao.deleteById(uid);
+            Assertions.fail();
+        }
+        catch(AuthDaoException authDaoException) {
```

Comment:
use assertThatThrownBy(() -> activeUserDao.deleteById(uid)).isInstanceOf(AuthDaoException)

NO need to use try catch
You can also check for the message, it should have a method'

Same below